docs/querying/basics.md

```diff
@@ -244,6 +244,8 @@ This modifier is disabled by default since it breaks the invariant that PromQL
 does not look ahead of the evaluation time for samples. It can be enabled by setting
 `--enable-feature=promql-at-modifier` flag. It will be enabled by default in the future.
 
+`start()` and `end()` can also be used as values for `@` modifier. Learn more about it [here](#pre-processors)
```

Comment:
"... for the `@` modifier."

Not a native speaker myself, but I'm pretty sure we should always use an article for the `@` modifier. This also applies to previously existing text above.